src/skins/vector/css/matrix-react-sdk/views/rooms/_EventTile.scss

```diff
@@ -94,7 +94,7 @@ limitations under the License.
  */
 .mx_EventTile_selected .mx_EventTile_line {
     border-left: $accent-color 5px solid;
-    padding-left: 60px;
+    padding-left: 100px;
```

Comment:
please let's not change the whole layout of the app just to provide the option of 12h timestamps ;)

See comments of https://github.com/matrix-org/matrix-react-sdk/pull/903/files#r117365391